aten/src/ATen/native/Activation.cpp

```diff
@@ -32,14 +33,47 @@ DEFINE_DISPATCH(leaky_relu_stub);
 DEFINE_DISPATCH(leaky_relu_backward_stub);
 
 Tensor hardtanh(const Tensor& self, Scalar min, Scalar max) {
+// #if defined(C10_MOBILE)
```

Comment:
Given that this are runtime ops, and not the ones which are folded/fused into conv/linear, you might want to just do this at clamp? https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Activation.cpp#L35. My impression is that in tracing, it will trace `clamp` and you may not see hardtanh at all. Same for relu's `threshold`, I presume.

Reply:
I wanted to do that but I felt that would be even more intrusive.  I might be wrong.